renderdoc/3rdparty/tinyexr/tinyexr.h

```diff
@@ -659,7 +659,8 @@ extern int LoadEXRFromMemory(float **out_rgba, int *width, int *height,
 #endif
 
 #if defined(TINYEXR_USE_MINIZ) && (TINYEXR_USE_MINIZ==1)
-#include <miniz.h>
+// RenderDoc change: we have included our own miniz already
+//#include <miniz.h>
```

Comment:
Instead of this RD-specific change could we use the `TINYEXR_USE_MINIZ` provided to include our own? It looks like this is inside the implementation macro so could be done in `tinyexr.cpp`.

Reply:
What is the error you hit? I quickly applied this PR's patch and `#define TINYEXR_USE_MINIZ 0` in tinyexr.cpp and it worked in a quick test since we're already including miniz.h there.

It looks in the header there's sections akin to:

```cpp
#if defined(TINYEXR_USE_MINIZ) && (TINYEXR_USE_MINIZ==1)
#elif defined(TINYEXR_USE_STB_ZLIB) && (TINYEXR_USE_STB_ZLIB==1)
#elif defined(TINYEXR_USE_NANOZLIB) && (TINYEXR_USE_NANOZLIB==1)
#else
  // call to compressBound / compress or uncompress
#endif
```

where `compressBound` etc in the `#else` case is expected to come from zlib, but miniz provides a #define to export that interface. I think the idea is you enable the `TINYEXR_USE` corresponding to which one you've provided, and if all are 0 then you need to have the zlib functions available somewhere at global scope. A little weird, but achieves the result we're after.

Reply:
Aaaah! Ok will try this now.